src/mixins/saveTokenSelectionMixin.js

```diff
@@ -73,9 +73,9 @@ export default {
         amount, isFrom,
       } = this.$route.query;
       const getToken = (token) => this.$store.getters['tokens/getAvailableTokens']().find(
-        (_token) => (
-          _token.is_ae && _token.symbol === token)
-          || (!_token.is_ae && _token.contract_id === token),
+        (_token) => (_token.is_ae
+          ? _token.symbol === token && _token.networkId === this.$store.state.networkId
+          : _token.contract_id === token),
```

Comment:
shouldn't we also check for network id here? could the same contract_id theoretically exist on two networks?

Reply:
I thought about it too, but is that a real case? I was thinking is like UUID, improbable to collide.
Anyway, I can change it.

Reply:
done